### ISSUE TITLE
Data loss bug with ctrl+a -- it's very easy to change all the numbers in a page, and there is no undo

### ISSUE THREAD

**Issue by lefth** [vipzhicheng/logseq-plugin-vim-shortcuts#52]
I was trying to select and copy a line. I don't remember the exact sequence of keys and/or selection, but I pressed ctrl+A twice. I may have also pressed Esc. What happened was that the whole document was selected, then all numbers were incremented by one. The change could not be undone. Math formulas were changed. Quantities were changed. URLs were modified. Paragraph references were modified. The change was not consistent, so subtracting 1 from the first number on each line might or might not fix it. (If I pressed ctrl+x, I would not know if the change was reversed correctly.) Undo did not work to restore the previous content. I am upset that my data was lost, however it was not that precious and I think I've restored the old values anyway (with some help from the backup files in `Logseq/logseq/version-files/local/pages`).

**Comment by Insufficient** [comment]
@lefth, I have experienced this as well. It looks like it is a feature because it replicates behaviour of Vim. 

I was able to disable it by going into the plugin settings JSON file (vertical dot menu -> Plugins -> Vim Shortcuts -> Open Settings > Edit settings.json) and changing the _increase_ and _decrease_ entries to null.

Example:
```
...
    "increase": null,
    "decrease": null,
...
```

**Comment by lefth** [comment]
This behavior is not like vim! The two differences are:

- the same key binding is not shared for select and modification
- the increment number operation creates an undo point

**Comment by Insufficient** [comment]
You're right - I guess the author is not able to replicate the exact behaviour but has mapped it that way because the keybinds are the same

**Comment by lefth** [comment]
I'm more concerned about the lack of undo. This could affect other operations as well.

**Comment by e-zz** [comment]
I am also a bit late to realize that I have invoked this behavior without noticing it.

It's safer to mute it in `system menu-keymap`. Just search for `vim increase` and delete the keybinding.
I also removed keys for `decrease` and `cut`

**Comment by vipzhicheng** [comment]
v0.2.0 has been released,  now it will only increase or decrease heading number of selected blocks. And you can disable these actions in the new Settings UI.

**Comment by lefth** [comment]
I had a quick test and it seems not to do anything on Mac.

I suppose the Logseq API still doesn't offer an API call to make an undo point that applies to the current page?

**Comment by vipzhicheng** [comment]
Yes, so what option I provided is that you can disable the keybindings if the actions are not needed or dangerous to you.
